app/uk/gov/hmrc/filetransmission/services/queue/TransmissionRequestWorkItemRepository.scala

```diff
@@ -63,7 +63,16 @@ class TransmissionRequestWorkItemRepository @Inject()(
   }
 
   def clearRequestQueue(): Future[Boolean] = {
-    mongoComponent.mongoConnector.db().collection[JSONCollection](name = "transmission-request").drop(failIfNotFound = false)
+    collection.drop(failIfNotFound = false)
+  }
+
+  def updateWorkItemBody(workItemId: BSONObjectID,
```

Comment:
I was thinking about renaming it to saying more explicitly that it updates only deliveryAttempts and nothing else.